week1/assignment/todo-tan/list.rb

```diff
@@ -0,0 +1,79 @@
+require_relative "item"
+
+class List
+  attr_accessor :name
+  attr_accessor :items
+
+  def initialize(name, items = [])
+    @name = name
+    @items = items
+  end
+
+  def add(item)
+    @items << item
+  end
+
+  def remove_at(index)
+    @items.delete_at(index)
+  end
+
+  def remove_by_name(name)
+
```

Comment:
for unimplemented methods, consider putting something like `raise "Not Yet Implemented"`